TileFormats/FeatureTable/schema/featureTable.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "featureTable.schema.json",
+    "title" : "Feature Table",
+    "type" : "object",
+    "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
+    "patternProperties" : {
```

Comment:
If a property is an object, does it have to have a `byteOffset` property?

For example, if a tile format wants to define a Feature Table with a global semantic like
```
"KEY" : {
   "some-property" : 0.0,
   "another-property" : [1, 2, 3]
}
```
Can it do so as the schema is written here?  (It might, I just don't know v4).

I suggest that we let the schema be flexible (and note this in the Feature Table spec) and rely on each tile format to specifically scope what is valid.  This will be forward-compatible as we and others add new tile formats.